syntax = "proto3";

service ReducerService {
    rpc RecievePartition(partitionRequest) returns (updated_centroids);
}

message partitionRequest {
    string partition = 1;
    string id = 2;
}
message updated_centroids{
    string updated_centroid = 1;
}